tests/conftest.py

```diff
@@ -525,3 +527,16 @@ def fn(exception=TransactionFailed, exc_text=None):
             assert exc_text in str(excinfo.value), (exc_text, excinfo.value)
 
     return fn
+
+
+@pytest.fixture(autouse=True)
```

Comment:
i'm not sure autouse fixtures are the best way to go about this, maybe add a pytest item collection hook? i don't remember exactly what the best practice is

Reply:
I went with `pytest_runtest_setup` from the docs [here](https://doc.pytest.org/en/latest/example/markers.html#custom-marker-and-command-line-option-to-control-test-runs).

Reply:
correction: that does not work, but `pytest_runtest_call` [does](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.hookspec.pytest_runtest_call).